test/TestGrains/StatelessWorkerGrain.cs

```diff
@@ -13,9 +10,11 @@
 
 namespace UnitTests.Grains
 {
-    [StatelessWorker(1)]
+    [StatelessWorker(ExpectedMaxLocalWorkers)]
     public class StatelessWorkerGrain : Grain, IStatelessWorkerGrain
     {
+        public const int ExpectedMaxLocalWorkers = 1;
```

Comment:
sure, I thought so too (it was that way just before submitting), it's just that I didn't want to reuse the same name as the `StatelessWorkerAttribute.MaxLocalWorkers` property, but it doesn't really matter, I'll update with your suggestion.